unittests/model_tests/neuron/test_synaptic_manager.py

```diff
@@ -113,13 +113,14 @@ def test_write_data_spec():
         additional_parameters={
             "splitter": SplitterAbstractPopulationVertexFixed()})
     proj_one_to_one_1 = p.Projection(
-        pre_pop, post_pop, p.OneToOneConnector(),
+        pre_pop, post_pop, p.FromListConnector([(i, i) for i in range(10)]),
```

Comment:
Why remove the check of a OneToOne and an AllToAll?

If wanted ADD a fromList or convert one of the OneToOne

Reply:
Undone changes - must have been testing something here!